build.gradle

```diff
@@ -1403,12 +1403,7 @@ project(':streams') {
   dependencies {
     compile project(':clients')
 
-    // this dependency should be removed after we unify data API
-    compile(project(':connect:json')) {
-      // this transitive dependency is not used in Streams, and it breaks SBT builds
-      exclude module: 'javax.ws.rs-api'
-    }
```

Comment:
This will break users who were relying on the transitive dependency. We may want to add a note to the release notes.